packages/trip-details/src/index.tsx

```diff
@@ -41,6 +42,19 @@ function dietaryLink(contents: ReactElement): ReactElement {
     </a>
   );
 }
+
+function CO2DescriptionLink(contents: ReactElement): ReactElement {
+  return (
+    <a
+      href="https://www.itf-oecd.org/sites/default/files/life-cycle-assessment-calculations-2020.xlsx"
+      rel="noopener noreferrer"
```

Comment:
Do we not want the itf to know we're using their report?

Reply:
ESlint wants those on there for security reasons if your target is a new window... I don't ask questions.

Reply:
ESlint wants those on there for security reasons if your target is a new window... I don't ask questions.